src/follower.py

```diff
@@ -73,21 +71,57 @@ def stop(self):
 
 
     """
-    Turn the wheels towards the last recognized object.
+    Turn the wheels and pans the camera towards the last recognized object.
     """
     def turn(self):
         self.convert_camera_angle()
-        self._fw.turn(self._turn_angle)
+        self.pan_camera()
+        
+        if self._turn_angle < 90:
+            self._fw.turn(self._turn_angle + self.camera_angle_offset)
+        elif self._turn_angle > 90:
+            self._fw.turn(self._turn_angle - self.camera_angle_offset)
 
 
     """
-    Converts the camera's angle scale to the same scale as the wheels.
+    Follows the tag by panning camera towards the same direction as the wheels.
+    """
+    def pan_camera(self):
+        increase_camera_offset = 0.5
+        if self._turn_angle < 90:
+            self.turn_camera_left(self._turn_angle)
+            self.camera_angle_offset = (90 - self._turn_angle)*increase_camera_offset
```

Comment:
Why are you dividing the offset by 2? Shouldn't the offset be equal to the angle we pan the camera not just half of it? Also do the angles increase or decrease going toward the right from the car's point of view? In this case, if your turn angle is 75, then your offset would be 15. Is that correct or should it be -15?